experimental/auth-logic/endorsement-release/input_policy_examples/github_actions_policy.auth_logic

```diff
@@ -0,0 +1,11 @@
+"GithubActionsPolicy" says {
+    "Provenance" canSay any_app
+        has_expected_hash_from(any_hash, "Provenance").
+
+    "Provenance" canSay any_app
+        has_builder(any_builder).
+
+    BuildPolicyAllowRelease(any_app, any_hash) :-
```

Comment:
What is the difference between predicates in SnakeCase vs lower_underscore  format? Should we just pick one?